server/src/projects/projects.service.ts

```diff
@@ -10,8 +11,13 @@ export class ProjectsService {
     @InjectModel(Project) private projectRepository: typeof Project,
   ) {}
 
-  create(createProjectDto: CreateProjectDto) {
-    return 'This action adds a new project';
+  async create(dto: CreateProjectDto) {
+    await this.projectRepository.create({
+      name: dto.name,
+      year: dto.year,
+    });
+
+    return CREATE_PROJECT;
```

Comment:
I think this message should returned as object 

 {  message: CREATE_PROJECT }